docs/src/pages/components/data-grid/events/events.md

```diff
@@ -8,7 +8,7 @@ title: Data Grid - Events
 
 ## Subscribing to events
```

Comment:
This page is not good. My idea is to remove the Pro badge from the page and move it to this section only. Then, add a new section only for the event props, e.g. `onCellClick`, explaining that they are like subscribing to events but with less granularity. For the catalog of events, I want to improve the docs generator to include a third column with the event prop that is mapped to each event name. I think that with our convention GRID_CELL_CLICK -> onCellClick I can discover the mappings. cc @oliviertassinari